sdk/python/kfp/cli/component.py

```diff
@@ -64,6 +64,10 @@
 _COMPONENT_ROOT_DIR = pathlib.Path('/usr/local/src/kfp/components')
 
 
+class ComponentBuildError(Exception):
```

Comment:
It seems like we don't need the `Exception` parameter.
```suggestion
class ComponentBuildError:
```

Reply:
Oh I see. Nevermind.